node/service/src/lib.rs

```diff
@@ -3,16 +3,26 @@
 #![warn(missing_docs)]
 #![warn(unused_extern_crates)]
 
+// // Don't allow both frequency and all-frequency-features so that we always have a good mainnet runtime
+// #[cfg(all(feature = "frequency", feature = "all-frequency-features"))]
+// compile_error!("feature \"frequency\" and feature \"all-frequency-features\" cannot be enabled at the same time");
+
+// // Don't allow both frequency-no-relay and all-frequency-features so that we always have a good mainnet runtime
```

Comment:
I think this was left on accident.

Reply:
This is what we decided to leave as a comment during pairing.  @wilwade Thoughts?

Reply:
We still want the compile errors, however they were not working. @rlaferla any thoughts on where to put them to make sure they work? Perhaps the cli/src/lib?

Reply:
Seems to work in `cli/build.rs`

Reply:
I have a solution but it will be in a separate issue/pr.